app/lib/shared/integrity.dart

```diff
@@ -456,7 +464,7 @@ class IntegrityChecker {
     if (pv.uploader == null) {
       yield 'PackageVersion "${pv.qualifiedVersionKey}" has no uploader.';
     } else {
-      yield* _checkUserValid(
+      yield* _checkAgentValid(
         pv.uploader!,
```

Comment:
We **critically** need to update documentation for this property.
All properties should specify what they contain, and what shape they have, when they may contain one thing and when they may contain something else..

Reading the model should provide sufficient documentation for writing the integrity check. It's always the model we look at when we write code -- we don't want to have to jump around the code to find other places.